osbs/api.py

```diff
@@ -713,6 +713,45 @@ def get_build_logs(self, build_id, follow=False, build_json=None, wait_if_missin
 
         return logs
 
+    def _parse_build_log_entry(self, entry):
+        items = entry.split()
+        if len(items) < 4:
+            logger.warning("This is not a valid build log entry: %s", entry)
```

Comment:
Ah, good call, otherwise, we'll potentially error out just a bit further down when trying to access the items by indices that aren't there.